docs/source/markdown/podman.1.md

```diff
@@ -274,58 +329,95 @@ the exit codes follow the `chroot` standard, see below:
 
 ## CONFIGURATION FILES
 
-**containers.conf** (`/usr/share/containers/containers.conf`, `/etc/containers/containers.conf`, `$HOME/.config/containers/containers.conf`)
+#### containers.conf
+
+**Path(s):**
+
+- `/usr/share/containers/containers.conf`
+- `/etc/containers/containers.conf`
+- `$HOME/.config/containers/containers.conf`
+
+**Info:**
+
+- Podman has builtin defaults for command line options. These defaults can be overridden using the `containers.conf` configuration files.
+
+- Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file.
+
+- Podman merges its builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrator's file, which overrides the distribution's file, which override the built-in defaults.
+
+- Podman uses builtin defaults if no `containers.conf` file is found.
```

Comment:
```suggestion
- Podman uses built-in defaults if there is no `containers.conf` file is found.
```

Reply:
Would you like it to be, "if there is no `containers.conf` file." ?